packages/vue/README.md

```diff
@@ -0,0 +1,364 @@
+    Disconnect MetaMask
+  </button>
+</template>
+```
+
+### `disconnectConnectedWallet`
+Function to disconnect the `connectedWallet`
+
+### Example usage
+```vue
+<script>
+import { useOnboard } from '@web3-onboard/vue'
+export default {
+  setup() {
+    const { disconnectConnectedWallet } = useOnboard()
+    return { disconnectConnectedWallet }
+  }
+}
+</script>
+
+<template>
+  <button type="button" @click="disconnectConnectedWallet">
+    Disconnect connectedWallet
+  </button>
+</template>
+```
+
+### `getChain`
+Function that returns the current chain a wallet is connected to
+
+### Example usage
+```vue
+<script>
+import { useOnboard } from '@web3-onboard/vue'
+export default {
+  setup() {
+    const { getChain } = useOnboard()
+    return { getChain }
+  }
+}
+</script>
+
+<template>
+  <span>MetaMask is connected to: {{getChain('MetaMask')}}</span>
+</template>
+```
+
+### `setChain`
+Function to set the chain of a wallet
+
+### Example usage
+```vue
+<script>
+import { useOnboard } from '@web3-onboard/vue'
+export default {
+  setup() {
+    const { setChain } = useOnboard()
+    const set = () => setChain({ wallet: 'MetaMask', chainId: '0x1' })
+    return { set }
+  }
+}
+</script>
+
+<template>
+  <button type="button" @click="set">
+    Set MetaMask chain to mainnet
+  </button>
+</template>
+```
+
+### `settingChain`
+Readonly boolean ref that tracks the status of setting the chain
+
+### Example usage
+```vue
+<script>
+import { useOnboard } from '@web3-onboard/vue'
+export default {
+  setup() {
+    const { settingChain } = useOnboard()
+    return { settingChain }
+  }
+}
+</script>
+
+<template>
+  <span v-if="settingChain">Setting chain...</span>
+</template>
+```
+
+### `wallets`
+Readonly ref that contains every wallet that has been connected
+
+### Example usage
+```vue
+<script>
+import { useOnboard } from '@web3-onboard/vue'
+export default {
+  setup() {
+    const { wallets } = useOnboard()
+    return { wallets }
+  }
+}
+```
+
+### `alreadyConnectedWallets`
+Readonly ref that contains every wallet that user connected to in the past; useful to reconnect wallets automatically after a reload
+
+### Example usage
+```
+vue
+<script>
+import { useOnboard } from '@web3-onboard/vue'
+export default {
+  setup() {
+    const { alreadyConnectedWallets } = useOnboard()
+    return { alreadyConnectedWallets }
+  }
+}
+</script>
+
+<template>
+  <div v-for="wallet in wallets">
+    <span>Label: {{wallet.label}}</span>
+  </div>
+</template>
+```
+
+### `lastConnectedTimestamp`
+Readonly ref that contains the last time that the user connected a wallet in milliseconds
+
+### Example usage
+```
+vue
+<script>
+import { useOnboard } from '@web3-onboard/vue'
+export default {
+  setup() {
+    const { lastConnectedTimestamp } = useOnboard()
+    return { lastConnectedTimestamp }
+  }
+}
+</script>
+
+<template>
+  <span>Your last connection timestamp was: {{ lastConnectedTimestamp }}</span>
+</template>
+```
```

Comment:
Are there any build/bundler challenges you ran into when using the Vue package?

Reply:
Nothing in particular Vue related, had some trouble with vite but it is already known (also how to workaround with polyfills)